### PR TITLE
Astt 11 graphical user interface

### PR DESCRIPTION
**Pull request by shusi98**
Run the GUI trough flask was a success  see running port number on the picture below
flask

track the tickcket here [ASTT-11](https://skaafrica.atlassian.net/jira/software/projects/ASTT/boards/153?selectedIssue=ASTT-11)

### CODE REVIEW

**Review by USilere — APPROVED** [review]

**Review by USilere — APPROVED** [review]
Looks good my side

**Review by AbednigoLethole — APPROVED** [review]
Looks good.

**Comment by shusi98** [comment]
couldn't merged because of conflicts please review and approve again

File: README.md [hunk 1]
```diff
@@ -2,59 +2,71 @@
...
+## **Running the component manager**
 
-## **Running the C++ Simulators**  
+* To run the component manager install the python requirements first
 
-* move to the simulators directory.
+``` pip3 install -r requirements.txt ```
 
-    ```cd src/canopen-simulator ```
+* run the asst component manager
```
Comment (mamkhari) [inline README.md]: What is asst?

**Review by mamkhari — COMMENTED** [review]
What is asst?

File: README.md (hunk 1 above)
Comment (shusi98) [reply README.md]: typing error it will be fixed

Comment (mamkhari) [reply README.md]: Alright, I thought it stood for something that I didn't know of as it has been used even here: https://github.com/AbednigoLethole/asst-cam-software

**Review by mamkhari — APPROVED** [review]
LGTM.

**Pull request merged**